public/styles/main.css

```diff
@@ -1,18 +1,58 @@
+/* GENERICS - project wide ------------------------ 
+- TO Maybe: once enough generics - abstract to it's own file?
+
+>> some defaults by probing screenshots
+- background-color: #f4f5fb
+- accent (button/active/item) color: #5670f1
+- sub text color: #c9cace
+- text color: #45494c
+- secondary color: #FFCC91
+*/
 *,*::before,*::after {
   box-sizing: border-box;
 }
 
+a, a:link, a:visited {
+  text-decoration: none;
+  color: #c9cace;
+
+  &:hover {
+    filter: brightness(115%); 
+  }
+}
+
+.active {
+  background-color: #5670f1 !important;
+}
+
+.content.content.header > .disabled,
+.content.content.content.main > .disabled,
+.side-bar > .disabled,
+.disabled {
+  filter: grayscale(100%) blur(3px) !important;
+  cursor: not-allowed;
+
+  &:hover{
+    background-color: transparent;
+    filter: blur(1px) !important;
+  }
+}
+
+/* END OF GENERICS ------------------- */
+
 body {
   margin: 0;
   padding: 0;
   display: grid;
-  font-family: system-ui, sans-serif;
+  background-color: #f4f5fb;
+  color: #45494c;
+  font-family: system-ui, Roboto;
```

Comment:
should this not be `Roboto` first? We'll always have `system-ui` on all systems

Reply:
as per slack conversation I've added roboto and fallback - system